Utilities/DataCompression/test/test_HuffmanCodec.cxx

```diff
@@ -20,27 +20,23 @@
 //* any purpose. It is provided "as is" without express or implied warranty. *
 //****************************************************************************
 
-//  @file   test_huffmancodec.cxx
+//  @file   test_HuffmanCodec.cxx
 //  @author Matthias Richter
-//  @since  2015-08-11
+//  @since  2016-08-11
 //  @brief  Test program for Huffman codec template class
 
-// Compilation: make sure variable BOOST_ROOT points to your boost installation
-/*
-   g++ --std=c++11 -g -ggdb -I$BOOST_ROOT/include -I../include -pthread -o test_huffmancodec test_huffmancodec.cxx
-*/
-
+#define BOOST_TEST_MODULE Utility test
+#define BOOST_TEST_MAIN
+#define BOOST_TEST_DYN_LINK
+#include <boost/test/unit_test.hpp>
 #include <iostream>
 #include <iomanip>
-#include <cstring>
 #include <vector>
 #include <bitset>
-#include <random> // std::exponential_distribution
-#include <cmath>  // std::exp
 #include <thread>
 #include <stdexcept>  // exeptions, runtime_error
-#include "DataCompression/dc_primitives.h"
-#include "DataCompression/HuffmanCodec.h"
+#include "../include/DataCompression/dc_primitives.h"
```

Comment:
Because there is no library in this module, the `include` directory is not automatically added by our cmake machinery, so it's needed in this special case.